tests/functional/aws-node-sdk/test/bucket/getBucketTagging.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+const async = require('async');
+
+const getConfig = require('../support/config');
+
+const bucket = 'gettaggingtestbucket';
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be  ' +
+            `${errors[expectedErr].code}, but got '${err.statusCode}'`);
+    }
+    cb();
+}
+
+describe('aws-sdk test get bucket tagging', () => {
+    let s3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        return done();
```

Comment:
Doesn't seem like we need done here

Reply:
done https://github.com/scality/cloudserver/pull/4481/commits/96157ec37c9e4398618eb993ff7e5fb92c4e8141#diff-5e6446c3e90586b538e1b00f9605b791ca303183acc5533411925d0e8be625d7R12